native-code/android/index.md

```diff
@@ -49,6 +49,16 @@ the code, building etc.
       * To build for 32-bit x86: use `target_cpu="x86"`
       * To build for 64-bit x64: use `target_cpu="x64"`
 
+     **NOTICE: Component Builds are not supported**
```

Comment:
Yeah, you're probably right. Let's wait for feedback from gn-dev@ before we decide on this.